tests/modules/authorize/lib/Auth/Process/AuthorizeTest.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Test for the authorize:Authorize authproc filter.
+ */
+
+namespace SimpleSAML\Test\Module\consent\Auth\Process;
```

Comment:
consent?

Reply:
Ooops! I was copying the namespace form another file. I ran into another problem, after moving the Authorize class into it's own file I get this error `Class 'SimpleSAML\Test\Module\authorize\Auth\Process\TestableAuthorize' not found`   I'm guessing the autoloader doesn't know how to load test classes that follow the module namespace convention. I'm not sure how to address that: Make `_autoload_modules.php` aware of the `SimpleSAML\Test` namespace? Add a dev autoloader to composer? Or am I completely off base and the cause it something else?

Reply:
I'd say the autoload-dev option would be the nicest way to fix this. I'm surprised we don't have that in place yet to be honest.

Reply:
I've added the `autoload-dev` option